firebase-perf/src/test/java/com/google/firebase/perf/config/ConfigurationConstantsTest.java

```diff
@@ -91,6 +91,7 @@ public void getInstance_SessionsCpuCaptureFrequencyForegroundMs_validateConstant
         SessionsCpuCaptureFrequencyForegroundMs.getInstance();
 
     assertThat(configFlag.getDefault()).isEqualTo(100L);
+    assertThat(configFlag.getDefaultOnRcFetchFail()).isEqualTo(300L);
```

Comment:
Would we have such asserts for the sampling rates?

Reply:
Added.